xcm/src/v3/traits.rs

```diff
@@ -212,6 +212,52 @@ impl From<SendError> for Error {
 
 pub type Result = result::Result<(), Error>;
 
+/*
+TODO: XCMv4
```

Comment:
I'm guessing the code here should be migrated to v4 when we get to creating it.

Reply:
Indeed.